src/SignalRServiceExtension/TriggerBindings/SignalRTriggerBindingProvider.cs

```diff
@@ -39,20 +38,54 @@ public Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext contex
             {
                 return Task.FromResult<ITriggerBinding>(null);
             }
-            var resolvedAttribute = GetParameterResolvedAttribute(attribute);
+            var resolvedAttribute = GetParameterResolvedAttribute(attribute, parameterInfo);
             ValidateSignalRTriggerAttributeBinding(resolvedAttribute);
             
             return Task.FromResult<ITriggerBinding>(new SignalRTriggerBinding(parameterInfo, resolvedAttribute, _dispatcher));
         }
 
-        private SignalRTriggerAttribute GetParameterResolvedAttribute(SignalRTriggerAttribute attribute)
+        internal SignalRTriggerAttribute GetParameterResolvedAttribute(SignalRTriggerAttribute attribute, ParameterInfo parameterInfo)
         {
             //TODO: AutoResolve more properties in attribute
+            var hubName = attribute.HubName;
+            var category = attribute.Category;
+            var @event = attribute.Event;
+            var parameterNames = attribute.ParameterNames ?? new string[0];
+
+            var method = (MethodInfo)parameterInfo.Member;
+            var parameters = method.GetParameters().Where(p => p.GetCustomAttribute<SignalRParameterAttribute>(false) != null);
+            var parameterNamesFromAttribute = new List<string>();
+            foreach (var parameter in parameters)
+            {
+                parameterNamesFromAttribute.Add(parameter.Name);
+            }
+
+            if (parameterNamesFromAttribute.Count != 0 && parameterNames.Length != 0)
+            {
+                throw new InvalidOperationException(
+                    $"{nameof(SignalRTriggerAttribute)}.{nameof(SignalRTriggerAttribute.ParameterNames)} and {nameof(SignalRParameterAttribute)} can not be set in the same Function.");
+            }
+
+            parameterNames = parameterNamesFromAttribute.Count != 0
```

Comment:
Should check `parameterNames` first? `parametersNames` should have higher priority than attributes.

Reply:
I don't want `parametersNames`  and attribute are set together. So, there's no priority issue.